cfgov/v1/util/migrations.py

```diff
@@ -57,3 +61,73 @@ def get_or_create_page(apps, page_cls_app, page_cls_name, title, slug,
     parent_page.save()
 
     return page
+
+
+def get_stream_data(page_or_revision, field_name):
+    """ Get the stream field data for a given field name on a page or a
+    revision """
+    try:
+        # If page_or_revision is a page, this will work
+        field = getattr(page_or_revision, field_name)
+        stream_data = field.stream_data
+    except AttributeError:
+        # Otherwise it will raise an Attribute error and we can assume
+        # page_or_revision is a revision
+        revision_content = json.loads(page_or_revision.content_json)
+        field = revision_content[field_name]
+        stream_data = json.loads(field)
+    return stream_data
+
+
+def set_stream_data(page_or_revision, field_name, stream_data):
+    """ Set the stream field data for a given field name on a page or a
+    revision """
+    try:
+        # If page_or_revision is a page, this will work
+        field = getattr(page_or_revision, field_name)
+        stream_block = field.stream_block
+        stream_value = StreamValue(stream_block, stream_data, is_lazy=True)
+        setattr(page_or_revision, field_name, stream_value)
+        page_or_revision.save()
+    except AttributeError:
+        # Otherwise it will raise an Attribute error and we can assume
+        # page_or_revision is a revision
+        revision_content = json.loads(page_or_revision.content_json)
+        revision_content[field_name] = json.dumps(stream_data)
+        page_or_revision.content_json = json.dumps(revision_content)
+        page_or_revision.save()
+
```

Comment:
I'm thinking something like this:

```
def _check_stream_data(page_or_revision, field_name):
	_is_page = True
    try:
        _field = getattr(page_or_revision, field_name)
    except AttributeError:
    	_field = json.loads(page_or_revision.content_json)
    	_is_page = False
    return _field, _is_page

def set_stream_data(page_or_revision, field_name, stream_data):
	field, is_page = _check_stream_data(page_or_revision, field_name)
    if is_page:
        stream_block = field.stream_block
        stream_value = StreamValue(stream_block, stream_data, is_lazy=True)
        setattr(page_or_revision, field_name, stream_value)
    else:
        field[field_name] = json.dumps(stream_data)
        page_or_revision.content_json = json.dumps(revision_content)
    page_or_revision.save()

def get_stream_data(page_or_revision, field_name):
	field, is_page = _check_stream_data(page_or_revision, field_name)
    if is_page:
        stream_data = field.stream_data
    else:
        stream_data = json.loads(field[field_name])
    return stream_data
```

Even here, I think one way to really think about this, is what if I later in my models added a field called 'content_json' to the page object?  Would this cause a breaking change later on?